README.md

```diff
@@ -25,17 +32,17 @@ The ruleset the passwords have to be validated against is:
 - There are no more than two repeating consecutive characters (like 'bbb' – two are allowed, but not three or more)
 - There is at least one upper-case character OR one special character [anything that is neither letter nor a number]
 
-## 2 - Script for checking passwords
+## Task 2 - Script for batch-validation of passwords
 
-We provided you with an SQL dump of 200 passwords. Write a CLI script in Node.js that reads those passwords from a DB and uses the API you created earlier to validate each of them. It should output a success message or the errors you received from the API for each of the responses.
+We provided you with an SQL dump of 200 passwords. Write a CLI script in Node.js that reads those passwords from a DB and uses the API you created earlier to validate each of them. It should output a success message or the errors you received from the API for each response.
 
-The script must set the `valid` field to 1 or 0 for valid and invalid passwords.
+The script must also set the `valid` field to 1 or 0 for valid and invalid passwords in the given MySQL table.
 
 Please provide and document a possibility to change the MySQL connection settings used by the script.
 
-## 3 (BONUS) - Checking for compromised passwords
+## Task 3 (BONUS) - Checking for compromised passwords
 
-Extend your script to check each password not only for validity but also if it has been compromised in the past. Please use the provided compromised password API for this.
+Extend the script you produced for Task 2 to include a check that uses a second API which tells you if passwords have been compromised in the past. This second API is already provided (see the next paragraph).
```

Comment:
Nitpick: if we need to suggest "see the next paragraph", maybe there's some else to improve in how this is explained.

We probably don't need a level-3 title "Using the compromised password API"

Consider something more or less like:

> (...) have been compromised in the past. 
>
> This second API is already implemented for you in this Docker image (...)

Reply:
Ok fixed